server_py/flatgov/common/management/commands/load_crec.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand
+from common.crec_data import crec_loader
+from bills.models import CommitteeDocument
+
+
+
+class Command(BaseCommand):
+    help = 'create cbo data.'
```

Comment:
This should say 'create committee data'. I think 'cbo' was copied from the CBO command.